binlog_streamer.go

```diff
@@ -183,6 +187,9 @@ func (s *BinlogStreamer) Run() {
 			// with empty GenericEvent structs.
 			// so there's no way to handle this for us.
 			continue
+		case *replication.XIDEvent, *replication.GTIDEvent:
+			s.lastResumableBinlogPosition.Pos = uint32(ev.Header.LogPos)
```

Comment:
Let's document in code comments what got us to this? 🙏

Reply:
can do 👍